wp-content/themes/csisnuclearnetwork/functions.php

```diff
@@ -172,10 +172,12 @@ function nuclearnetwork_register_styles() {
 
 	$theme_version = wp_get_theme()->get( 'Version' );
 
-	wp_enqueue_style( 'nuclearnetwork-fonts', 'https://fonts.googleapis.com/css2?family=Lato:wght@400;700&family=Merriweather:wght@400;700&family=Public+Sans:wght@700&display=swap', array(), null );
+	wp_enqueue_style( 'nuclearnetwork-fonts', 'https://fonts.googleapis.com/css2?family=Lato:wght@400;700&family=PT+Sans:wght@400;700&display=swap', array(), null );
 
 	wp_enqueue_style( 'nuclearnetwork-style', get_stylesheet_directory_uri() . '/style.min.css', array(), $theme_version );
 
+	wp_enqueue_style( 'nuclearnetwork-carousel', get_stylesheet_directory_uri() . '/node_modules/@splidejs/splide/dist/css/splide-core.min.css', array(), $theme_version );
```

Comment:
Unless you updated the Gulp config to include this, this file won't be included in the push from your local machine to GitHub because it's contained in `node_modules`, and it won't be included when GH Action deploys the site to the server either. You can either copy the contents of the CSS file into a file in your `assets/scss/vendors` folder, or use the [jsDeliver CDN](https://www.jsdelivr.com/package/npm/@splidejs/splide?path=dist%2Fcss) link.